src/test/java/com/google/plantasy/TimingPropertiesTest.java

```diff
@@ -275,21 +234,16 @@ public void giveUserQuizTakenPoints_notScoreProperty() {
         user.setProperty("quiz_timestamp", 159430944365L);
         datastore.put(user);
 
-        boolean actual = false;
-        try{
-            actual = timing_properties_test.giveUserQuizTakenPoints(true, user, datastore);
-        } catch(NullPointerException e) {
-            user.setProperty("score", 20);
-        }
-        Assert.assertEquals(user.getProperty("score"), 20);
+        boolean actual = timing_properties_test.giveUserQuizTakenPoints(true, user, datastore);
+        user.setProperty("score", 20);
     }
 
     @Test
     //Test if user was given points if their initial value was not zero
     public void giveUserQuizTakenPoints_actuallyAddedPoints_notStartAtZero(){
         QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-
+        
```

Comment:
More pesky trailing whitespace here

Reply:
Yeah I got rid of these in the next commit